docs/general/manage-tokens/oasis-wallets/browser-extension.mdx

```diff
@@ -2,8 +2,8 @@
 description: The Oasis Foundation-managed non-custodial browser extension wallet
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
 
 # ROSE Wallet - Browser Extension
```

Comment:
@matevz Kaja added screenshots, but text and flows would need to be updated too. Do you plan to deduplicate most of the text and flows with Web Wallet? The main difference is layout size and ledger flow has an extra step: ledger2.png